### DISCUSSION TITLE
Wrong compilation

### DISCUSSION THREAD

**Discussion by laoo** [mbitsnbites/vasm-mirror#20] [General]
After more research it turns out that the code is actually valid.

> 
> ```	text
> label btst.b #1,label(pc,d0)
> ```
> should produce
> 
> ```083b000100fe```
> 
> but it produces
> 
> ```083b000100fc```
> 
> instead.
> 
> 
> 
>

**Comment by StarWolf3000** [comment]
This repository is not an official one, it is not affiliated with the vasm developer. We only host a copy of the source code. Therefore we cannot provide any help, issues should be reported to Frank Wille at sun.hasenbraten.de. You can find contact info here: http://www.compilers.de/contact_eng.html

**Comment by laoo** [comment]
@StarWolf3000 thanks!
